scripts/check_lint.sh

```diff
@@ -15,7 +15,7 @@ if [ -z "$lint_path" ]; then
 fi
 
 DOCKER_CMD=${DOCKER_CMD:-docker}
-GOLANGCI_LINT_IMAGE="golangci/golangci-lint:v1.31.0"
+GOLANGCI_LINT_IMAGE="canislabs/golangci-lint:latest"
```

Comment:
Could we please revert this change?

Reply:
So that's one of the changes I was referring to when I mentioned that this was going to be complicated.  The linter won't work unless the indy_vdr lib is accessible so I had to create another docker image with that library.  

I welcome any suggestions for alternative!

Reply:
@pfeairheller One improvement would be to make GOLANGCI_LINT_IMAGE a script argument, with the default being "golangci/golangci-lint:v1.31.0"

`GOLANGCI_LINT_IMAGE=${GOLANGCI_LINT_IMAGE:-golangci/golangci-lint:v1.31.0}`

then where you call `make lint` in GH Actions:
`GOLANGCI_LINT_IMAGE=<alternate image> make lint`

Reply:
If the indy_vdr lib is fetchable, it could also be possible to create a docker image as a CI step
FROM golangci/golangci-lint

Reply:
Unfortunately, that lib is not yet fetchable.  I like your first suggestion and will take a shot at it.  I'll also open an issue with indy-vdr to publish their lib and see if I can help get that moving.

Reply:
Next build will have the GOLANGCI_LINT_IMAGE passed as an env variable so the Indy VDR can specify the modified docker image with the lib.